content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -502,6 +507,49 @@
         }
     }
 
+    /**
+     * Scrolls the browser when the URI fragment is changed to the item of the container Accordion component that corresponds to the deep link in the URL fragment,
+       and displays its content.
+     * This method fixes the issue existent with Chrome and related browsers, which are just scrolling to the item without displaying its content.
+     */
+    function onHashChange() {
+        if (location.hash && location.hash !== "#") {
+            var anchorLocation = decodeURIComponent(location.hash);
+            var anchorElement = document.querySelector(anchorLocation);
+            if (anchorElement && anchorElement.classList.contains("cmp-accordion__item")) {
+                var anchorElementButton = document.querySelector(anchorLocation + "-button");
```

Comment:
Why not reusing `expandItem` and `collapseItem` to open and close the corresponding item?

Reply:
Or, even bettter, `toggle` if not `getItemExpanded` (it includes datalayer calls)